api/types/server.go

```diff
@@ -178,9 +180,9 @@ func (s *ServerV2) Expiry() time.Time {
 	return s.Metadata.Expiry()
 }
 
-// SetPublicAddr sets the public address this cluster can be reached at.
-func (s *ServerV2) SetPublicAddr(addr string) {
-	s.Spec.PublicAddr = addr
+// SetPublicAddrs sets the public proxy addresses where this server can be reached.
+func (s *ServerV2) SetPublicAddrs(addrs []string) {
+	s.Spec.PublicAddrs = addrs
```

Comment:
Keep back-compat in mind. If we just start leaving the old `PublicAddr` field blank, that might have unintended effects when dealing with older entities.

We should continue to ensure that `PublicAddr` is filled out for a full major version cycle.  IDK if this is the place to do it, but aught to happen somewhere.